book/tut5_topography.md

```diff
@@ -0,0 +1,258 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# 3D Topography 🏔️
+
+In this tutorial, let's use PyGMT to create 3D perspective plots of Digital Elevation
+Models (DEM) over Mars (the planet) and Antarctica (the continent)!
+
+🔖 References:
+
+- https://www.generic-mapping-tools.org/remote-datasets/mars-relief.html
+- https://github.com/andrebelem/PlanetaryMaps
+- {cite:t}`NeumannMarsOrbiterLaser2003`
+
+
+```{code-cell}
+import pygmt
+import rioxarray
+import rioxarray.merge
+```
+
+# 0️⃣ Mars relief data
+
+First, we'll load the Mars Orbiter Laser Altimeter (MOLA) dataset using
+[`pygmt.datasets.load_mars_relief`](https://www.pygmt.org/v0.13.0/api/generated/pygmt.datasets.load_mars_relief.html).
+The following command will load the MOLA dataset into an
+[`xarray.DataArray`](https://docs.xarray.dev/en/v2024.09.0/generated/xarray.DataArray.html)
+grid, and we'll set the `resolution` parameter to `01d` (1 arc-degree) for now.
+
+```{code-cell}
+da_mars = pygmt.datasets.load_mars_relief(resolution="01d")
+da_mars
+```
+
+## 2D map view
+
+Here we can create a map of the entire Martian surface, using a
+[Mollweide projection](https://www.pygmt.org/v0.13.0/projections/misc/misc_mollweide.html).
+To get a reddish hue, we'll use a
+[colormap](https://docs.generic-mapping-tools.org/6.5/reference/cpts.html)
+called `SCM/managua` which comes with a soft hinge
```

Comment:
Hm. For me, it works to directly give the name of a Scientific colourmap (without the folder "SCM"). We are also doing this in the PyGMT examples.

```suggestion
called `managua` which comes with a soft hinge
```

Reply:
Yes, there's backwards compatibility with just `managua` (see https://github.com/GenericMappingTools/gmt/pull/6446#issuecomment-1126711471). The `SCM/` prefix is intended for better recognition that this is a Scientific Colour Map, see comment at https://docs.generic-mapping-tools.org/6.5/reference/cpts.html:

> The color maps are subdivided into a number of sections relating to their source. This avoids name clashes and improves recognision of the color maps as well as their authors

Reply:
Thanks for pointing out this GMT PR! Hm, maybe we should then do this for all tutorials consistently.